easybuild/easyblocks/l/lammps.py

```diff
@@ -216,20 +224,21 @@ def configure_step(self, **kwargs):
         if self.cfg['kokkos']:
 
             if self.toolchain.options.get('openmp', None):
-                self.cfg.update('configopts', '-DKOKKOS_ENABLE_OPENMP=yes')
+                self.cfg.update('configopts', '-DKokkos_ENABLE_OPENMP=yes')
```

Comment:
Latest versions of LAMMPS enforce the `Kokkos` in the cmake defs but they are backwards compatible so safe to swap out